lib/DeviceManagerPlugin.ts

```diff
@@ -381,4 +320,53 @@ export class DeviceManagerPlugin extends Plugin {
 
     return request;
   }
+
+  private getDefaultConfig () {
```

Comment:
If you really want to extract this then you could put it in a separate file for example